examples/best_practices/resource_default_namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
Move resource manifest to examples/best_practices/resources/ ? So the user can import the best practices directly from here?